docs/upgrade.html

```diff
@@ -124,6 +124,11 @@ <h5><a id="upgrade_101_notable" href="#upgrade_101_notable">Notable changes in 1
 <ul>
     <li>Restored binary compatibility of AdminClient's Options classes (e.g. CreateTopicsOptions, DeleteTopicsOptions, etc.) with
         0.11.0.x. Binary (but not source) compatibility had been broken inadvertently in 1.0.0.</li>
+    <li>Kafka Streams is more robust against broker communication errors. Instead of stopping the Kafka Streams client with a fatal exception,
+	Kafka Streams tries to self-heal and reconnect to the cluster. Using the new <code>AdminClient</code> you have also more control how often
+	Kafka Streams should retry and configure timeouts fine grained (instead of hard coded retries as in older version).</li>
```

Comment:
nit: "fine grained" -> fine-grained

Reply:
The new `AdminClient` is only added starting 1.1.0 right?

Reply:
Ups. Wrong section.